packages/node_modules/@ciscospark/react-component-activity-system-message/src/index.js

```diff
@@ -8,32 +8,44 @@ import styles from './styles.css';
 
 export const SYSTEM_MESSAGE_VERB_TOMBSTONE = `tombstone`;
 export const SYSTEM_MESSAGE_VERB_CREATE = `create`;
-export const SYSTEM_MESSAGE_VERBS = [SYSTEM_MESSAGE_VERB_CREATE, SYSTEM_MESSAGE_VERB_TOMBSTONE];
+export const SYSTEM_MESSAGE_VERB_UPDATE = `update`;
+export const SYSTEM_MESSAGE_VERBS = [SYSTEM_MESSAGE_VERB_CREATE, SYSTEM_MESSAGE_VERB_TOMBSTONE, SYSTEM_MESSAGE_VERB_UPDATE];
 
 export default function ActivitySystemMessage(props) {
   const {
+    callData,
     isSelf,
     name,
     timestamp,
     verb
   } = props;
 
   let systemMessage;
-  if (verb === SYSTEM_MESSAGE_VERB_CREATE) {
+  switch (verb) {
+  case SYSTEM_MESSAGE_VERB_CREATE: {
     if (isSelf) {
       systemMessage = <FormattedMessage {...messages.youCreate} />;
     }
     else {
       systemMessage = <FormattedMessage {...messages.someoneCreate} values={{name}} />;
     }
+    break;
   }
-  else if (verb === SYSTEM_MESSAGE_VERB_TOMBSTONE) {
+  case SYSTEM_MESSAGE_VERB_TOMBSTONE: {
     if (isSelf) {
       systemMessage = <FormattedMessage {...messages.youDelete} />;
     }
     else {
       systemMessage = <FormattedMessage {...messages.someoneDelete} values={{name}} />;
     }
+    break;
+  }
+  case SYSTEM_MESSAGE_VERB_UPDATE: {
+    systemMessage = <FormattedMessage {...messages[callData.status]} values={{...callData.callInfo}} />;
+    break;
+  }
+  default:
+    return null;
```

Comment:
It would be great to be able to do this at the selector and just pass it as a param, but I still can't figure out a good way to get i18n in the selector.